stdlib/email/base64mime.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Callable
+
+def header_length(bytearray: bytes) -> int: ...
+def header_encode(header_bytes: str | bytes, charset: str = ...) -> str: ...
+def body_encode(s: bytes, maxlinelen: int = ..., eol: str = ...) -> str: ...
+def decode(string: str | bytes) -> bytes: ...
+
+body_decode: Callable[[str | bytes], bytes]
+decodestring: Callable[[str | bytes], bytes]
```

Comment:
just copies of `decode` for backwards compatibility.

Reply:
Does this not work?

```suggestion
body_decode = decode
decodestring = decode
```

Reply:
(I haven't tried it!)

Reply:
worth a shot!

Reply:
nice that works fine, good spot 😄

Reply:
Hooray 😀